pyproject.toml

```diff
@@ -30,16 +30,16 @@ celery = "^5.2.6"
 fastparquet = "^0.8.1"
 Fiona = "^1.8.21"
 geojson = "^2.5.0"
-geopandas = "^0.10.2"
+geopandas = "^0.12.0"
 geopy = "^2.2.0"
 gunicorn = "^20.1.0"
-ibis-framework = {extras = ["postgres"], version = "^3.2.0"}
+ibis-framework = "^7.2.0"
```

Comment:
Were you able to test all calls to the database using this version? Perhaps to be able to use it you will have to repair many other deprecated ones.

Reply:
do we have tests for these calls?